fem/hybridization.hpp

```diff
@@ -64,6 +64,12 @@ class Hybridization
    FiniteElementSpace *fes, *c_fes;
    BilinearFormIntegrator *c_bfi;
 
+   /// Set of constraint boundary face integrators to be applied.
+   Array<BilinearFormIntegrator*> boundary_constraint_integs;
+   Array<Array<int>*>             boundary_constraint_integs_marker;
+   /// Indicates if the boundary_constraint_integs integrators are owned externally
+   int extern_bdr_constr_integs;
```

Comment:
It's a minor point but I would prefer a boolean here since that seems to be how this variable is used.

Reply:
Me too, I just followed the convention in MFEM to use ints for no good reason 😄